test/e2e/validation_test.go

```diff
@@ -56,13 +55,18 @@ func getInstanceID(providerID string) string {
 }
 
 // getInstanceIDsOfNode returns the instanceIDs of all the Windows nodes created
-func getInstanceIDsOfNode(kubeclient kubernetes.Interface) ([]string, error) {
-	nodes, err := kubeclient.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: wmc.WindowsOSLabel})
+func (tc *testContext) getInstanceIDsOfNode() ([]string, error) {
```

Comment:
Given you are changing this and all the call sites, please correct it to `getInstanceIDsOfNodes`